iac/cloudinit/cloud-init.yml

```diff
@@ -67,4 +67,5 @@ runcmd:
   - ["systemctl", "daemon-reload" ]
   - ["systemctl", "enable", "minecraft"]
   - ["systemctl", "start", "--no-block", "minecraft"]
+  - ["cd minecraft && make"]
```

Comment:
this should not be like this.
 - the directory is wrong
 - the make has no default target (which means the default is stopping the server)

Reply:
I tried the directory in my computer and it only worked like thissss